magento/magento2
# Fix usage of EAV cache type

### Description (*)

This PR fixes usage of EAV cache type in some places

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. _Different frontend type can be set to EAV cache type without defining additional eav frontend type introduced by `configured_eav_cache` virtual type_
UPD: looks like it's bigger problem I see many other classes do not use `\Magento\Eav\Model\Cache\Type` and they can't be fixed due to `Magento\Framework\App\CacheInterface` and `Magento\Framework\Cache\FrontendInterface` incompatibilities.
It should be figured out and fixed in different PR/issue.
2. Class `\Magento\Catalog\Plugin\Model\ResourceModel\Config` checks if  EAV cache enabled but store data in default cache type. This PR fixes missed configuration

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#32322: Fix usage of EAV cache type
